Packs/Azure-Enrichment-Remediation/Playbooks/Azure_-_Network_Security_Group_Remediation.yml

```
@@ -828,6 +840,8 @@ tasks:
               applyIfEmpty: {}
               defaultValue: {}
             operator: SetIfEmpty
+      using:
+        simple: ${inputs.instance_name}
```

```suggestion
        simple: ${inputs.InstanceName}
```
You might want to change it to title case like the other inputs.